wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WireOutput.kt

```diff
@@ -41,11 +42,12 @@ abstract class WireOutput {
   abstract fun toTarget(): Target
 
   /** Configure the project's graph to compile the files emitted. */
-  abstract fun applyToProject(
+  open fun applyToProject(
     project: Project,
     wireTask: TaskProvider<WireTask>?,
     kotlin: Boolean
-  )
+  ) {
+  }
```

Comment:
I wouldn't mind keep that as abstract, and let the implementer no-op it.

Reply:
fixed